### DISCUSSION TITLE
insert into database, other column is fine but email column getting null value in supabase

### DISCUSSION THREAD

**Discussion by rozabond** [supabase/supabase#22215] [Questions]
Here is my insert method
![Bug](https://github.com/supabase/supabase/assets/67621191/787d16d1-a197-4479-b481-bde450a3898e)
Here is what i get, all inserted excepted the email column
![Bug2](https://github.com/supabase/supabase/assets/67621191/bb0efa6b-61ed-494b-bdb6-90e3b7e20d0b)

**Comment by GaryAustin1** [comment]
Any errors?  Any chance the id exists already and insert failed?  Is formdata good?  Any trigger on the table that might modify email?  Your format seems correct.
